src/vmm/src/arch/aarch64/regs.rs

```diff
@@ -109,6 +109,10 @@ arm64_sys_reg!(ID_AA64MMFR2_EL1, 3, 0, 0, 7, 2);
 // EL0 Virtual Timer Registers
 arm64_sys_reg!(KVM_REG_ARM_TIMER_CNT, 3, 3, 14, 3, 2);
 
+/// Vector lengths pseudo-register
+pub const KVM_REG_ARM64_SVE_VLS: u64 =
```

Comment:
maybe we can add a todo to remove this once https://github.com/rust-vmm/kvm-bindings/pull/89 is resolved?

Reply:
added